hawc/apps/assessment/api.py

```diff
@@ -445,3 +456,36 @@ def get(self, request, casrn: str, format=None):
             tasks.get_dsstox_details.delay(casrn)
 
         return Response(data)
+
+
+class BlogViewset(viewsets.ReadOnlyModelViewSet):
+    model = models.Blog
+    serializer_class = serializers.BlogSerializer
+    pagination_class = None
+
+    def get_queryset(self):
+        return self.model.objects.filter(published=True)
+
+
+class LogViewset(viewsets.ReadOnlyModelViewSet):
```

Comment:
some refactoring of this one - would this be easy to do?

- I'd make the list view `/assessment/log/` return system logs if IsAdminUser
- I'd make the detail view `/assessment/log/1/` return logs for assessment 1. 

Alternatively, you could add a new detail action 'logs' on `AssessmentViewset` that returns assessment logs. That's probably easier to do and would require less customization

Reply:
Update - looks like you already did the suggestion! I'd just make the first bullet then. I'd use the `ListModelMixin` here with a standard `viewsets.ViewSet`, and then just handle the list view. We probably don't need a detail view for this one, and I'd just set the permission_class to (IsAdminUser,); users can access the assessment logs via a different API